templates_ark/1_Segment_Image_Data.ipynb

```diff
@@ -41,6 +41,25 @@
     "If you're interested in directly interfacing with Google Drive, consult the documentation [here](https://ark-analysis.readthedocs.io/en/latest/_rtd/google_docs_usage.html)."
```

Comment:
<p>I think we can consolidate the different paths here and in <code>base_dir</code> below to have a single top-level directory defined with base_dir, and everything else defined based on that </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/685/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>